apps/api/afbcore/views/authtoken.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import status
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+# NOTE: The passwordless "token/code" is only ever set to is_active=False
+# in two places (both in drfpasswordless/utils.py):
+#
+#   1. authenticate_by_token() -- not called within passwordless
+#   2. validate_token_age() -- called by validate_token() in passwordless
+#
+# That's why magic links stay valid even after they're used. They stay
+# valid and (currently) can be used N times until they expire.
+# See: PASSWORDLESS_TOKEN_EXPIRE_TIME in settings.py.
+
+
+class LogoutView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def post(self, request, version=None):
+        try:
+            request.user.auth_token.delete()
```

Comment:
The request user will have an `auth_token` in all (normal) cases thanks to `IsAuthenticated`. 

We're taking an approach with this codebase to avoid being defensive where and when we can. If auth_token isn't set the end result is fail-safe. The user is logged out either way.